app/containers/Home/WithHomeMarkUp.jsx

```diff
@@ -0,0 +1,29 @@
+// @flow
+import React, { Component } from 'react'
+import styles from './Home.scss'
+import neonLogo from '../../images/neon-logo-redesign.png'
+
+type Props = {
+  children: React$Node,
+  renderNavigation?: Function
+}
+
+export default class WithHomeMarkUp extends Component<Props> {
```

Comment:
What do you think about renaming this to something like `HomeLayout` instead?

Reply:
Much better... The `with` naming convention is supposed to be reserved for denoting HOC.